woocommerce/woocommerce
# Updates Mailchimp branding in setup wizard

### All Submissions:


### Changes proposed in this Pull Request:
In the **Recommended** step in the **WooCommerce setup wizard**, the tile background color is changed to gold ( `#ffe01b` ), the monkey logo avatar image updated, and mentions of "MailChimp" in the text and link changed to "Mailchimp" in line with their recently [updated brand guidelines](https://mailchimp.com/about/brand-assets/).

Closes #22513.

### How to test the changes in this Pull Request:

1. Check out this branch in your local environment
2. Open the Setup Wizard and navigate to the **Recommended** step – `admin.php?page=wc-setup&step=recommended`
3. Check that the Mailchimp for WooCommerce branding and information looks the same as in the _After_ screenshot shown below

### Screenshot:
**Before:**
![Before](https://user-images.githubusercontent.com/6031418/51480989-37835100-1d9b-11e9-864d-dcc3714b482e.png)

**After:**
![After](https://user-images.githubusercontent.com/6031418/51480972-29cdcb80-1d9b-11e9-92c2-853d8a3133ac.png)


### Changelog entry

- Updates Mailchimp tile background color
- Removes outdated Mailchimp avatar image
- Adds updated Mailchimp avatar image
- Styles all mentions of "MailChimp" in copy to "Mailchimp"
